models/activities/action.go

```diff
@@ -394,7 +394,7 @@ func (a *Action) GetIssueInfos() []string {
 // with the action.
 func (a *Action) GetIssueTitle() string {
 	index, _ := strconv.ParseInt(a.GetIssueInfos()[0], 10, 64)
-	issue, err := issues_model.GetIssueByIndex(a.RepoID, index)
+	issue, err := issues_model.GetIssueByIndex(db.DefaultContext, a.RepoID, index)
```

Comment:
:eyes: I'm not particularly fond of using `db.DefaultContext` outside the tests.

Reply:
Because this function will be invoked in template, so I think maybe we can keep it in this function and further refactor could be other PRs.

Reply:
Could you maybe add a comment indicating this?